app/models/story.rb

```diff
@@ -15,10 +15,16 @@ class Story
   index(created_at: 1)
   index(updated_at: 1)
 
+  field :age_confirm
+  field :guardian_consent
+
   accepts_nested_attributes_for :ore_aggregation
 
   validates_associated :ore_aggregation
 
+  validates :age_confirm, acceptance: { accept: ['true', 1] } unless :guardian_consent
```

Comment:
These should be showing validation error messages if not accepted, even without JS validation.